decidim-admin/app/helpers/decidim/admin/attributes_display_helper.rb

```diff
@@ -31,7 +31,7 @@ module AttributesDisplayHelper
       # attrs - a list of N attributes of the `record`.
       def display_for(record, *attrs)
         attrs.map do |attr|
-          if record.column_for_attribute(attr).type == :hstore
+          if record.column_for_attribute(attr).type == :jsonb
```

Comment:
Since we're fixing this, is there anyway to assert this without having to couple it to a column type?